builds/e2e/templates/nested-parent-vm-setup.yaml

```diff
@@ -1,29 +1,11 @@
 parameters:
   upstream.protocol: 'amqp'
-  testRunnerCount: '1'
+  test.l4DeploymentFileName: ""
```

Comment:
Terminology question -- this file is named nested-_parent_-setup.yaml and internally has variables for the L4 deployment file and L3 IP Address, so _parent_ in this context refers to a L5 device, but not a L4 device? This is out of scope for this task but in the future referring to this as the nested-L5-setup.yaml or nested-top-layer-vm-setup.yaml would provide more clarity.

Reply:
The parent in this context refers to either L5 or L4 devices. It actually does have jobs for both of these parent devices.